tasks/deployment/safety-module-recovery.ts

```diff
@@ -27,7 +28,7 @@ hardhatTask('deploy:safety-module-recovery', 'Deploy the Safety Module recovery
       safetyModuleRecovery,
       safetyModuleRecoveryProxyAdmin,
     } = await deploySafetyModuleRecovery(args);
-    console.log(`New Safety Module implementation deployed to: ${safetyModuleNewImpl.address}`);
-    console.log(`Safety Module recovery contract deployed to: ${safetyModuleRecovery.address}`);
-    console.log(`Safety Module recovery proxy admin deployed to: ${safetyModuleRecoveryProxyAdmin.address}`);
+    log(`New Safety Module implementation deployed to: ${safetyModuleNewImpl.address}`);
+    log(`Safety Module recovery contract deployed to: ${safetyModuleRecovery.address}`);
+    log(`Safety Module recovery proxy admin deployed to: ${safetyModuleRecoveryProxyAdmin.address}`);
```

Comment:
Were these meant to be `console.log` statements? I can change them back if you'd prefer that (since you need to set `DEPLOYMENT_LOGS=true` to see these statements currently).

Reply:
`DEPLOYMENT_LOGS` defaults to `true` unless `config.isHardhat() && !config.FORK_MAINNET` so we're probably good with `log()`